File: API/eProject.API/eProject.DataAccess/Models/Response/RequestResult.cs
using eProject.DataAccess.Models.Enum;
using System;
using System.Collections.Generic;
using System.Text;

namespace eProject.DataAccess.Models.Response
{
    public class RequestResult
    {
        public StatusCode StatusCode { get; set; }
        public string Content { get; set; }
    }


}

File: Group1_ProjectSEM4/Source code/Source code Andoird application/Android/app/src/main/java/com/group1/project4/models/RequestResult.java
package com.group1.project4.models;

public class RequestResult {
    private int statusCode;
    private String content;

    public int getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(int statusCode) {
        this.statusCode = statusCode;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public RequestResult(int statusCode, String content) {
        this.statusCode = statusCode;
        this.content = content;
    }
}
